jenkins/helper/test_launch_controller.py

```diff
@@ -776,21 +776,21 @@ def generate_crash_report(self):
         if IS_MAC:
             move_files = True
             system_corefiles = list(Path('/cores').glob(core_pattern))
-        files_unsorted = list(core_dir.glob(core_pattern)) + system_corefiles
+        files_unsorted = list(core_dir.glob(core_pattern))
+        if files_unsorted is None:
+            files_unsorted = []
+        files_unsorted += system_corefiles
         if len(files_unsorted) == 0 or core_max_count <= 0:
             print(f'Coredumps are not collected: {str(len(files_unsorted))} coredumps found; coredumps max limit to collect is {str(core_max_count)}!')
             return
-        files = files_unsorted.copy().sort(key=get_file_size, reverse=True)
```

Comment:
It seems that sorting by file size is now removed.It seems the sorting wasn't necessary at all, but can you confirm this is in fact unneeded.

Reply:
yes, it was part of a previous approach to first delete bigger corefiles.